Doc/library/ssl.rst

```diff
@@ -976,6 +1006,26 @@ Constants
 
    .. versionadded:: 3.6
 
+.. class:: TLSVersion
+
+   :class:`enum.IntEnum` collection SSL and TLS versions for
+   :attr:`SSLContext.highest_version` and :attr:`SSLContext.lowest_version`.
+
+   .. versionadded:: 3.7
+
+.. attribute:: TLSVersion.MINIMUM_SUPPORTED
+.. attribute:: TLSVersion.MAXIMUM_SUPPORTED
+
+   The minimum or maximum supported SSL or TLS version. The value of the
+   enum value does not reflect the actual minimum or maximum version.
```

Comment:
I don't understand what the second sentence means.

Reply:
I tried to explain that  ``MAXIMUM_SUPPORTED == TLSv1_2`` or ``MAXIMUM_SUPPORTED == TLSv1_3`` doesn't give any meaningful result. You cannot use the value of the ``MAXIMUM_SUPPORTED`` constant to figure out what the actual maximum supported TLS version is. The values are just arbitrary values.

Reply:
Maybe something like "this is a magic constant, it will not be equal to the value of the constant for the highest protocol"?